app/controllers/likes_controller.rb

```diff
@@ -0,0 +1,37 @@
+class LikesController < ApplicationController
+  before_action :find_event
+  before_action :find_like, only: [:destroy]
+
+  def create
+    if already_liked?
+      flash[:notice] = "You can't like more than once"
```

Comment:
Hmm I think if you already liked an event, and you press Like again, I would expect only to remove the like, like a boolean

Reply:
with like, it's to avoid that a user can like more than once on a single event.

Reply:
so the logic would be, if `already_liked?` destroy the like

Reply:
yes.

Reply:
I made the new push and made the correction